general/clients/index.md

```diff
@@ -12,6 +12,25 @@ Clients connect your devices to your Jellyfin server and let you view your conte
 
 Do you have a client that interfaces with Jellyfin and want to see it listed here? Please [submit a pull request](https://github.com/jellyfin/jellyfin-docs)!
 
+## Browsers
+
+Our goal is to provide support for these browsers. 
+
+ - last 2 Firefox versions
+ - last 2 Chrome versions
+ - last 2 ChromeAndroid versions
+ - last 2 Safari versions
+ - last 2 iOS versions
```

Comment:
Should probably be grouped with the Safari line and changed to "last 2 versions of Safari of MacOS and iOS"

Reply:
What about older tablets? Would they need back support like chrome does?

Reply:
If they're still getting iOS updates, they should be good, I think? I don't use Apple products, so I'm not fully sure what their support policy is.

Reply:
I refactored the layout a little, tell me what you think